cmd/manager/main.go

```diff
@@ -49,6 +52,18 @@ func doMain() int {
 		return 1
 	}
 
+	numOfClients := defaultNumberOfClients
+	envNumOfClients, found := os.LookupEnv(envVarBundlesSyncerNumberOfClients)
+
+	if found {
+		if value, err := strconv.Atoi(envNumOfClients); err != nil {
+			log.Info(fmt.Sprintf("Failed to convert environment variable '%s', value: %s, err: %s. Using default %d.",
```

Comment:
I believe this should be logged as an error (log.Error instead of log.Info) since if the user of the repo assigned a non-convertible value to the env var then their intentions were to set it to something.

Reply:
done